<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindowClass</class>
 <widget class="QDialog" name="SettingsWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="container">
     <property name="styleSheet">
      <string notr="true">QFrame {
	background-color: #282a36;
	color: #f8f8f2;
	border-radius: 15px;
	/*font: 12pt &quot;Ubuntu&quot;;*/
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 9pt;</string>
      </property>
      <property name="text">
       <string>Pomo Time (mm:ss):</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbPMinutes">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>30</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="suffix">
       <string>m</string>
      </property>
      <property name="prefix">
       <string/>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbPSeconds">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>30</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="suffix">
       <string>s</string>
      </property>
      <property name="maximum">
       <number>59</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbRSeconds">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>70</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="suffix">
       <string>s</string>
      </property>
      <property name="maximum">
       <number>59</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 9pt;</string>
      </property>
      <property name="text">
       <string>Rest Time (mm:ss):</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbRMinutes">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>70</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="suffix">
       <string>m</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 9pt;</string>
      </property>
      <property name="text">
       <string>Hide top bar</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbAutoHideTitleBar">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox::indicator:checked {
	background-color: #ff79c6;
	border: none;
}
QCheckBox::indicator:unchecked{
	background-color: rgba(189, 147, 249, 145);
	border: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSave">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	width: 100px;
    height: 25px;
    color: rgb(151, 159, 200);
    background-color: rgb(68, 71, 90);
    font-size: 18px;
    border-radius: 12px;
}
    
QPushButton:hover:!pressed {
    background-color: rgb(151, 159, 200);
    color:  rgb(68, 71, 90);
}</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCancel">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>150</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	width: 100px;
    height: 25px;
    color: rgb(151, 159, 200);
    background-color: rgb(68, 71, 90);
    font-size: 18px;
    border-radius: 12px;
}
    
QPushButton:hover:!pressed {
    background-color: rgb(151, 159, 200);
    color:  rgb(68, 71, 90);
}</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>110</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 9pt;</string>
      </property>
      <property name="text">
       <string>Auto start rest time</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbAutoStartRestTime">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>110</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox::indicator:checked {
	background-color: #ff79c6;
	border: none;
}
QCheckBox::indicator:unchecked{
	background-color: rgba(189, 147, 249, 145);
	border: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbSmallPomo">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>110</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox::indicator:checked {
	background-color: #ff79c6;
	border: none;
}
QCheckBox::indicator:unchecked{
	background-color: rgba(189, 147, 249, 145);
	border: none;
}</string>
      </property>
      <property name="text">
       <string>Small pomo</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>110</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 9pt;</string>
      </property>
      <property name="text">
       <string>Small pomo</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
